apps/kitchen-sink/src/App.tsx

```diff
@@ -21,16 +21,25 @@ import FirstScript from "./ensemble/scripts/test.js?raw";
 import EnsembleConfig from "./ensemble/config.yaml";
 // Theme
 import ThemeYAML from "./ensemble/theme.yaml";
+import DarkThemeYAML from "./ensemble/dark.yaml";
 
 import "./App.css";
 
 const testApp: ApplicationDTO = {
   id: "test",
   name: "My App",
-  theme: {
-    id: "theme",
-    content: String(ThemeYAML),
-  },
+  theme: [
```

Comment:
Let's add support for a new `themes` property that can take multiple while continuing to support old `theme` property. This is an important principle when developing the framework that we maintain backwards compatibility.